.github/workflows/ALL-PHPUnit.yml

```diff
@@ -19,25 +31,19 @@ jobs:
           - "8.3"
         pgsql-version:
           - "13"
+          - "16"
```

Comment:
I'm more curious than concerned about this, but why the jump from Postgresql 13 to 16? What happens if I wanted to test with 14 or 15?

Reply:
For the automated testing we check with the highest and the lowest supported versions. However, in repos where we automate building the docker images, we build images for all versions in between. 

You can see that in the build docker workflows:
- [TripalCultivate Base Images](https://github.com/TripalCultivate/TripalCultivate/blob/4.x/.github/workflows/MAIN-buildDocker.yml#L21-L103)
- [TripalDocker images](https://github.com/tripal/tripal/blob/4.x/.github/workflows/MAIN-buildDocker.yml#L21-L103)
- [TripalDocker Drupal images](https://github.com/tripal/tripal/blob/4.x/.github/workflows/SCHEDULE-buildDrupalDocker.yml#L20-L102)

We don't need to automate building of the phenotype/genotype/germplasm builds because they are not used by others really 🤷‍♀️  We could I guess but currently don't.